### PR TITLE
Fix plugin download issue for StatefulSet chart

### PR DESCRIPTION
**Pull request by haljoh**
Downloading of plugins fail since the plugin init container is not allowed to write the downloaded plugin into the download directory. If setting values for plugins: runAsUser and runAsGroup to 0 the download works. But, when SonarQube container starts up it doesn't have the permissions to move the plugins in to the correct folder. 

This fix solve the issue so the default user-id 1000 can download the plugin and SonarQube container then have the permissions to move the downloaded files. This issue has been fixed in the deployment.yaml file but must have been missed for the stateful set.

### CODE REVIEW

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]
Hi @haljoh ,

thanks for your contribution. upon review i fully agree that the pod security context is missing from the stateful set. can you just add your change to the Chart.yaml description as well as the CHANGELOG.md and then we can merge your contribution

File: charts/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -21,10 +21,8 @@ annotations:
     - name: support
       url: https://community.sonarsource.com/
   artifacthub.io/changes: |
-    - kind: added
-      description: "added link to community support forum"
-    - kind: changed
-      description: "Use liveness endpoint instead of helth endpoint for liveness probe"
+    - kind: fixed
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/Chart.yaml:24]: The previous changes have not been released yet, so please keep this list additive

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]

File: charts/sonarqube/Chart.yaml (hunk 1 above)
Comment (haljoh) [reply charts/sonarqube/Chart.yaml:24]: Ahh, ok, shall I then also keep chart version as 1.1.10? I created a 1.1.11 version for my change. Should I also change that back to version 1.1.10?

Comment (tobias-trabelsi-sonarsource) [reply charts/sonarqube/Chart.yaml:24]: nah the version increment to 1.1.11 (nice version btw :D) is correct

Comment (haljoh) [reply charts/sonarqube/Chart.yaml:28]: ok, I have added back the changes to the Chart.yaml file

**Comment by tobias-trabelsi-sonarsource** [comment]
The diff looks good to me now. sadly the git information seem off. could you try to squash your commits and rebase to master?

**Review by tobias-trabelsi-sonarsource — APPROVED** [review]

**Pull request merged**